runtime/lua/vim/treesitter/dev.lua

```diff
@@ -506,7 +523,10 @@ function M.inspect_tree(opts)
     buffer = buf,
     once = true,
     callback = function()
-      close_win(w)
+      -- close all tree windows
+      vim.iter(pairs(vim.fn.win_findbuf(b))):each(function(_, window)
+        close_win(window)
```

Comment:
nit: since we are no longer doing find() , or modifying the iterable in other ways, each() is actually more verbose than just using pairs().

```suggestion
      for _, window in pairs(vim.fn.win_findbuf(b)) do
        close_win(window)
```